finalPrep/annotations.js

```diff
@@ -2,41 +2,72 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+
+//Create a dynamic function with multiple (5) parameters
 function buildABear(name, age, fur, clothes, specialPower) {
+//Declare a variable (*greeting*) and assign a value that is a string containing the parameter *name* interpolated in.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//Declare a variable (*demographics) and assign the value to an array containing the (elements? parameters? not sure) *name* and *age*.
   var demographics = [name, age];
+//Declare a variable (*powerSaying*) and assign the value to a string containing the parameter *specialPower* concatenated in.
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//Create an object (*buildBear*). Begin by using the keyword `var` followed by the object's name (*builtBear*).
   var builtBear = {
+//Add key (*basicInfo*) value (*demographics*) pair to object. Value is a varible used above.
     basicInfo: demographics,
+//Add another key value pair to object. Value is a parameter from above.
     clothes: clothes,
+//Add another key value pair to object. Value is a parameter from above.
     exterior: fur,
+//Add another key value pair to object. Value is assigned here.
     cost: 49.99,
+//Add another key value pair to object. Value is an array of varibales from above and a string
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+//Add anotehr key value pair to object. Value is a Boolean.
     isCuddly: true,
+//Close of the object (*builtBear*)
   };
-
+//Return the builtBear string value when function is called.
   return builtBear
+//Close the function
 }
 
+//Add arguments to the dynamic function. This is where we submit specific arguments to the parameters.
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+//Add new arguements to the dynamic function
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
+//Create a dynamic function with 3 parameters (num1, num2, range)
 function fizzBuzz(num1, num2, range) {
+//Create a loop. Begin with the keyword `for`. Initialize (var = 0), set the condition (i <= range), and add increment expression (i++).
   for (var i = 0; i <= range; i++) {
+//Begin conditional if/ else if/else statement. Set the condition between parenthesis (i % num1 === 0 && i % num2 === 0).
```

Comment:
Put a little research into the `%` operator, and discuss with some classmates- I think this is excellent practice in technical explanation.